Doc/whatsnew/3.10.rst

```diff
@@ -105,11 +105,12 @@ Removed
 =======
```

Comment:
Even if it's unrelated to this PR, can you please add a "Porting to Python 3.10" section after the Removed section?